test/config/integration/server_xds.lds.yaml

```diff
@@ -0,0 +1,19 @@
+versionInfo: "0"
+resources:
+- "@type": type.googleapis.com/envoy.api.v2.Listener
+  name: listener_0
+  address:
+    socketAddress:
+      address: {{ ntop_ip_loopback_address }}
+      portValue: 0
+  filterChains:
+  - filters:
+    - name: http_connection_manager
+      config:
+        codecType: HTTP2
+        drainTimeout: 5s
+        statPrefix: router
+        rds:
+          routeConfigName: route_config_0
+          configSource: { path: {{ rds_json_path }} }
+        httpFilters: [{ name: router, config: { deprecatedV1: true }}]
```

Comment:
from json to yaml? and user configs have to mention these weird strings
`"@type": type.googleapis.com/envoy.api.v2.Listener` ? :(